app.js

```diff
@@ -87,7 +92,7 @@ router.post('/account', ratelimit({
 }), async ctx => {
     if (!creatorKeyJson) {
         console.warn('ACCOUNT_CREATOR_KEY is not set up, cannot create accounts.');
-        ctx.throw(404);
+        ctx.throw(500, 'Service misconfigured; account creation is not available.');
```

Comment:
@vgrichina This felt more like a `500` than ` 404` since expected functionality is broken without this environment variable but to the REST client a `404` seems confusing in this context, since the request is a POST that creates a new subject... happy to revert if this is an incorrect interpretation here